hknweb/utils.py

```diff
@@ -36,3 +37,20 @@ def get_rand_photo(width=400):
     with open(find("animal_photo_urls.txt")) as f:
         urls = f.readlines()
     return urls[randint(0, len(urls) - 1)].strip() + "?w=" + str(width)
+
+# date and time
+
+def get_semester(date):
+    """ Returns a string representation of the semester of this timezone object.
+        Assumes that there are only spring and fall semesters, separated at 07/01.
+        Example: "spring 2020" """
+    season = "spring" if date.month < 7 else "fall"
+    return "{} {}".format(season, date.year)
+
+def get_semester_bounds(date):
+    """ Returns the two dates that bound the current semester.
+        Assumes that there are only spring and fall semesters, separated at 07/01. """
```

Comment:
Summer semester?

Reply:
We don't have candidates in the summer

Reply:
We do have summer course survey data, however.

I’d at least add a comment specifying this was intended for candidate events, and not for the other uses of the Semester model we were working on.

Reply:
Also, maybe in the future we'd like to have custom/admin control over this date since it's a rigid `07/01`. Let's make an issue for this and for the point that James mentioned?

Reply:
I don't see why we need that, it's not like this date will change. If it does it won't change very often so keeping it hard coded should be fine

Reply:
Added comment